packages/amplication-server/src/core/workspace/workspace.service.ts

```diff
@@ -191,10 +191,12 @@ export class WorkspaceService {
   async createWorkspace(
     accountId: string,
     args: Prisma.WorkspaceCreateArgs,
+    isNewUserInitialWorkspace = false,
```

Comment:
@mulygottlieb, you can pass this parameter last, so you will not need to send it from the resolver

Reply:
There are optional args after so it's not possible to pass it last as much as I know